index.bs

```diff
@@ -1472,12 +1472,19 @@ interface WebTransportError : DOMException {
   constructor(optional WebTransportErrorInit init = {});
 
   readonly attribute octet? applicationProtocolCode;
+  readonly attribute WebTransportErrorSource source;
 };
 
 dictionary WebTransportErrorInit {
```

Comment:
We're probably going to be asked by the TAG to add `source` to `WebTransportErrorInit`. We might as well do it now, unless it will complicate matters.

Reply:
We don't have a plan to introduce a feature that initiates a sesson/connection level error. Is it reasonable to allow to create such an error?

Reply:
While I could see some marginal benefit of not being able to fake this value, I think the ability to shim errors is generally considered a good thing.

Reply:
> We don't have a plan to introduce a feature that initiates a sesson/connection level error.

Hmm, does this mean we could split `"stream"` into `"client"` and `"server"` to solve https://github.com/w3c/webtransport/issues/317?